test/recipes/controls/aws_parallelcluster_install/directories_spec.rb

```diff
@@ -0,0 +1,21 @@
+control 'setup_dirs' do
+  title 'Setup of basic directories'
+
+  base_dir = "/opt/parallelcluster"
+  dirs = [ base_dir, "#{base_dir}/sources", "#{base_dir}/scripts", "#{base_dir}/licenses", "#{base_dir}/configs", "#{base_dir}/shared" ]
+  dirs.each do |path|
+    describe directory(path) do
+      it { should exist }
+    end
+  end
+end
+
+control 'log_dir' do
```

Comment:
Same as above. In this case, using a speaking name would allow us to understand the behavior by just reading the list of controls in the kitchen config file.